src/main/java/com/sap/ordermanagergreen/service/GraphService.java

```diff
@@ -31,7 +27,25 @@ public class GraphService {
     @Autowired
     private IOrderRepository orderRepository;
     @Autowired
-    private MongoTemplate mongoTemplate;
+    public MongoTemplate mongoTemplate;
+
+    public List<TopEmployeeDTO> getTopEmployee() {
+
+        Aggregation aggregation = newAggregation(
+                match(Criteria.where("auditData.createDate").gte(LocalDate.now().minusMonths(3))),
+                match(Criteria.where("orderStatus").is(OrderStatus.DONE)),
+                group("employee").count().as("countOfDeliveredOrders"),
+                project("countOfDeliveredOrders").and("_id").as("employee"),
```

Comment:
The .as("employee") should be the same name as the parameter in TopEmployeeDTO
so you need change user to employee in TopEmployeeDTO
since system do the mapping between mongo result and DTO by name

please change and update me by result

thanks